common/thanos/templates/thanos.yaml

```diff
@@ -136,6 +136,20 @@ spec:
             {{- end }}
             {{- end }}
   {{- if $.Values.deployWholeThanos }}
+              args:
+              - query
+              - --grpc-address=0.0.0.0:10901
+              - --http-address=0.0.0.0:10902
+              - --log.level={{ default "info" $.Values.query.logLevel }}
+              - --query.auto-downsampling
+              - --query.replica-label=prometheus_replica
+              - --endpoint=thanos-kubernetes-thanos-kubernetes-store:10901
+              - --endpoint=prometheus-kubernetes:10901
+              - --web.external-prefix=/thanos
+              - --web.route-prefix=/thanos
+              {{- if $.Values.query.requestLog.enabled }}
+              - --request.logging-config-file=/conf/query/request_log.yaml
+              {{- end }}
```

Comment:
We need to whole overwrite directive to be able to enable the requestLog, am I getting this correctly?

Reply:
added missing configmap volume and mount